### PR TITLE
New field "manually-approve" for contact table

### PR DESCRIPTION
**Pull request by annando**
We had some places in the code where we used that field already but had to fetch it via the APContact table.

### CODE REVIEW

File: src/Model/Contact.php [hunk 1]
```diff
@@ -1851,9 +1851,8 @@ private static function updateFromProbeArray(int $id, array $ret)
 			$ret['prv'] = false;
 			$ret['contact-type'] = $ret['account-type'];
 			if ($ret['contact-type'] == User::ACCOUNT_TYPE_COMMUNITY) {
-				$apcontact = APContact::getByURL($ret['url'], false);
-				if (isset($apcontact['manually-approve'])) {
-					$ret['forum'] = (bool)!$apcontact['manually-approve'];
+				if (isset($ret['manually-approve'])) {
```
Comment (MrPetovan) [inline src/Model/Contact.php:1854]: Please simplify the conditional to a single level:
```php
if ($ret['contact-type'] == User::ACCOUNT_TYPE_COMMUNITY && isset($ret['manually-approve'])) {
```

File: src/Model/Contact.php [hunk 2]
```diff
@@ -2123,9 +2122,8 @@ public static function createFromProbe(array $user, $url, $interactive = false,
 
 		$pending = false;
 		if ($protocol == Protocol::ACTIVITYPUB) {
-			$apcontact = APContact::getByURL($ret['url'], false);
-			if (isset($apcontact['manually-approve'])) {
-				$pending = (bool)$apcontact['manually-approve'];
+			if (isset($ret['manually-approve'])) {
```
Comment (MrPetovan) [inline src/Model/Contact.php:2125]: Please simplify the conditional to a single level:
```php
if ($protocol == Protocol::ACTIVITYPUB && isset($ret['manually-approve'])) {
```

File: src/Model/Contact.php (hunk 1 above)
Comment (annando) [reply src/Model/Contact.php]: Done

File: src/Model/Contact.php (hunk 2 above)
Comment (annando) [reply src/Model/Contact.php]: Done!

**Pull request merged**